app/code/Magento/Catalog/Api/Data/CategoryInterface.php

```diff
@@ -143,6 +143,15 @@ public function setLevel($level);
     public function getChildren();
 
     /**
+     * Set category children
+     *
+     * @param string $children
+     * @return $this
+     */
+    public function setChildren($children);
```

Comment:
Unfortunately, we are not able to add new methods to the existing interfaces marked as api according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)